src/App.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from "prop-types";
+
+const todoList = [
+  { id: 1, title: "Add new task" },
+  { id: 2, title: "Edit" },
+  { id: 3, title: "Task done" },
+];
+
+Heading.propTypes = {
+  children: PropTypes.node.isRequired, // Validate children
+};
+
+List.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+export default function App() {
+  return (
+    <div>
+      <Heading>Todo List</Heading>
+      <UnorderedList />
```

Comment:
Componentization is very important and you have choosen well to do that for the unordered list.

The `Heading` component, on the other hand, is just rendering `h1`, so `h1`could be used interchangeably. Here are motivations for keeping it and removing it:
- We do create components like this when it improves the semantic and we know that it will evolve soon. Heading improves by giving a proper name, and will probably evolve soon by adding styles or links.
- The downside is that you can't use that same component if you need a level-two heading in the future - you would need a `Heading2`. In the end, you would be replicating HTML's existing elements one by one.

Reply:
It's up to you to keep the `Heading` component or not. As mentioned above, there are reasons for both, so I'm really OK with whatever you choose.

Reply:
Thank you, for clarification. I choose to leave Heading, but I decided to rename Heading to HeadingH1, I think it is more appropriate.